host/src/components/FeaturedMistakes.jsx

```diff
@@ -0,0 +1,162 @@
+    const resetMistakes = sortedMistakes.map((mistake, index) => {
+      if (index < 3){
+        handleOnSelectMistake(mistake.answer, true);
+        return {...mistake, isSelected: true};
+      }
+      return {...mistake, isSelected: false};
+    })
+    setSortedMistakes(resetMistakes);
+  };
+
+  const handleModeChange = (event) => {
+    if (event.target.value === 'A') {
+      resetMistakesToTop3();
+      setIsTop3Mode(true);
+    } else {
+      setIsTop3Mode(false);
+    }
+  };
+
+  const handleSelectMistake = (index) => {
+    handleOnSelectMistake(sortedMistakes[index].answer, false);
+    setSortedMistakes((prev) => {
+      const newMistakes = [...prev];
+      newMistakes[index].isSelected = !newMistakes[index].isSelected;
+      return newMistakes;
+    });
+  };
+
+  useEffect(() => {
+    setSortedMistakes(sortMistakes(shortAnswerResponses, totalAnswers));
+  }, [shortAnswerResponses, totalAnswers]);
+  return(
+    <Paper className={classes.background} elevation={0}>
+        <Typography className={classes.title}>{title}</Typography>
+        <Typography className={classes.subtitle}>{subtitle}</Typography>
+        <RadioGroup defaultValue="A" onChange={handleModeChange}>
+          <FormControlLabel 
+            className={classes.radioLabel} 
+            value="A" 
+            control={<Radio className={classes.radioButton} />} 
+            label={radioButtonText1} 
+          />
+          <FormControlLabel 
+            className={classes.radioLabel} 
+            value="B" 
+            control={<Radio className={classes.radioButton} />} 
+            label={radioButtonText2} 
+          />
+        </RadioGroup>
+        {sortedMistakes.length > 0 
+          ? <Box sx={{display: 'flex', flexDirection: 'column', alignItems: 'flex-start', justifyContent: 'flex-start', gap: 10, width: '100%'}}>
+            {sortedMistakes.map((mistake, index) => {
+                return <MistakeSelector 
+                  key={index} 
+                  mistakeText={mistake.answer} 
+                  mistakePercent={mistake.percent} 
+                  isTop3Mode={isTop3Mode} 
+                  isSelected={mistake.isSelected} 
+                  mistakeIndex={index}
+                  handleSelectMistake={handleSelectMistake} 
+                  style={{width:'100%'}}  
+                />
+              })}
+            </Box>
+          : <Box sx={{width: '100%'}}>
+              <Typography className={classes.subtitle} style={{fontStyle: 'italic', textAlign: 'center'}}>
+                Student responses will appear here
+              </Typography>
+            </Box>
+        }
+      
+    </Paper>
+  );
+};
+
+const useStyles = makeStyles(({
+  background: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    borderRadius: '24px',
+    padding: `16px`,
+    backgroundColor: 'rgba(0,0,0,0)', 
+    // boxShadow: '0px 8px 16px -4px rgba(92, 118, 145, 0.40)',
```

Comment:
Most of these comments have been left in intentionally because they will be uncommented for `host_v2`. That being said, I get that it's confusing, so I'm not going to be lazy and have removed the comments from our code.